pkgs/development/tools/build-managers/gup/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchgit, lib, python, which }:
+let
+  version = "0.5.1";
+  src = fetchgit {
+    url = "https://github.com/gfxmonk/gup.git";
+    rev = "f185052e2177ed5e46720e6c6cfb529b96cd17e2";
+    sha256 = "c2e27cdba2231017ceb4868f58f5c3b224be0491b81558b4e59bb08a952ad1a5";
+  };
+in
+import ./build.nix
```

Comment:
It's a bit strange because it's separated into local (default.nix) and upstream (build.nix) parts. The build.nix file is copied verbatim from https://github.com/gfxmonk/gup/blob/master/nix/gup-python.nix, because the upstream author (me) uses nix for development as well. This way the same build expression can be used for both development and in nixpkgs (aside from `src` which will usually differ between the two, and `meta` which is unnecessary in development).